src/main/default/lwc/caseAssistFlow/caseAssistFlow.js

```diff
@@ -52,6 +52,9 @@ export default class CaseAssistFlow extends LightningElement {
         console.error(err);
       }
     }, 500);
+
+    // Debounce function to not send an event every letter typed.
+    this.debounceTicketUpdate = debounce(this.sendTicketFieldUpdated, 200);
```

Comment:
Out of curiosity, why is the ticket update delay set to 200ms, while the search one is 500ms?

Reply:
Good question, it IS an arbitraty choice, but we wanted to track as many case updates as possible I thought, while we wanted to limit the rate of queries to the actual model?

That's my logic anyway.

Reply:
Personally, I would set both debounce with the same delay, mainly because it makes sense to me that both trigger at the same time (almost). But, honestly, this is also an arbitrary choice and your guess is as good as mine.

Reply:
Yea I think you are correct, I will change it to 500ms as well 👍